app/javascript/app/webauthn.js

```diff
@@ -15,10 +15,12 @@ const longToByteArray = long => new Uint8Array(8).map(() => {
   return byte;
 });
 
-const extractCredentials = credentials => credentials.split(',').map(credential => ({
-  type: 'public-key',
-  id: base64ToArrayBuffer(credential),
-}));
+const extractCredentials = credentials => {
+  return credentials.split(',').filter(element => element !== "").map(credential => ({
```

Comment:
`split` leaves an empty string and that turns into an empty `credential` object. Windows can handle `[]` for `excludeCredentials` but not the other thing. Is there a better way to do this?

Reply:
Maybe it's just time for the function to one be a one-liner anymore?

```
if (!credentials) { // empty string check
  return [];
}
return credentials.split(',').map( .... )
```

Reply:
Yeah, I like that better. Thanks Zach.